content/en/docs/user-journeys/users/application-developer/foundational.md

```diff
@@ -53,6 +53,33 @@ Minikube can be installed locally, and runs a simple, single-node Kubernetes clu
 
 You can get basic information about your cluster with the commands `kubectl cluster-info` and `kubectl get nodes`. However, to get a good idea of what's really going on, you need to deploy an application to your cluster. This is covered in the next section.
 
+#### microk8s
+
+On Linux, *microk8s* might be a good alternative to Minikube for a
+local install of Kubernetes. Microk8s keeps up with the current
+Kubernetes, and automatically updates to the latest point release for
+your convenience. With microk8s you don't to install a hypervisor or
+launch a virtual machine, and installation completes within ~30
+seconds.
+
+* {{< link text="Install microk8s" url="https://microk8s.io/" >}}.
+
+After you install microk8s, you can use its tab-completion
+functionality. All microk8s commands start with `microk8s.`. Type
+`microk8s.` (with the period) and then use the tab key to see a list
+of available commands.
+
+It also includes commands to enable Kubernetes subsystems. For example:
```

Comment:
Maybe "You can also use microk8s commands to enable...", sorry I know you already made one modification here.